test/metabase/query_processor_test/date_time_zone_functions_test.clj

```diff
@@ -734,7 +734,9 @@
                                  "2022-10-03T00:00:00Z")))))      ; 2022-10-03T00:00:00Z
         (testing "hour under a day"
           (mt/with-temporary-setting-values [driver/report-timezone "Atlantic/Cape_Verde"]
-            (is (partial= {:second 82800 :minute 1380 :hour 23 :day 1}
+            (is (partial= (if (driver/supports? driver/*driver* :set-timezone)
+                            {:second 82800 :minute 1380 :hour 23 :day 1}
+                            {:second 82800 :minute 1380 :hour 23 :day 0})
```

Comment:
SQL server doesn't support set-timezone, so this change verifies that setting the report-timezone doesn't have an effect.